src/tests/ui/text_input.ts

```diff
@@ -1,5 +1,6 @@
 import { ExpectedConditions as until, ElementFinder } from 'protractor';
```

Comment:
you can import Key from protractor
import { ExpectedConditions as until, ElementFinder, Key } from 'protractor';